src/modExcelToXML.bas

```
@@ -0,0 +1,169 @@
+Attribute VB_Name = "modExcelToXML"
+Option Explicit
+
+Const ms As Double = 0.000000011574
+
+Sub excelToXML()
```

The error handling method used here seems like it will ignore all errors. I'm not sure if this is a good idea. What's your thoughts about this?

---

No this is not a good idea, I have implemented your nice error handler into this with last commit 916a66f30fe7319ec0c08ff1f8c8de869f1d88e6